friendly/src/main/java/com/google/sps/servlets/GroupServlet.java

```diff
@@ -100,14 +102,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     response.getWriter().println("valid"); 
     members.add(ownerEmail);
+    // Construct a new list from the set constucted from elements 
+    // of the original list 
+    List<String> nonDuplicateMembers  = members.stream() 
+                                    .distinct() 
```

Comment:
I think the standard should be 1 additional tab space from the original line, when the line is too long

Reply:
Noted, thanks for pointing out!